src/botservice/azext_bot/.vscode/tags

```diff
@@ -0,0 +1,478 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
Does this file need to be checked-in to the repo?
What's it for?

Reply:
no it doesn't . i will add my .vscode folder to gitignore so that this doesnt get in by mistake